rm-main/src/ui/tabs/torrents/bottom_stats.rs

```diff
@@ -29,6 +47,13 @@ impl Component for BottomStats {
                 text = format!("󰋊 {free_space} | {text}")
             }
 
+            let table_manager = &*self.table_manager.lock().unwrap();
+            let state = table_manager.table.borrow();
+            if let (Some(current), total) = (state.state.borrow().selected(), state.get_len()) {
+                let current_idx = current + 1;
+                text = format!("{current_idx}/{total} | {text}")
```

Comment:
this is redundant, cause we are already showing the total amount of torrents (next to the file icon), could you move current_idx there?